README.md

```diff
@@ -1,23 +1,23 @@
-# exa
-Override `ls` and `tree` commands to use [exa](https://github.com/ogham/exa) instead
+# eza
+Override `ls` and `tree` commands to use [eza](https://github.com/eza-community/eza) instead
 
 ## Install
 Using the [:zap: Zap](https://www.zapzsh.org/) minimal zsh plugin manager, add the following to your `.zshrc`
 
 ```sh
 # Install plugin
-plug "zap-zsh/exa"
+plug "zap-zsh/eza"
 ```
 
 ## ⚠️ Special Attention
-As mentioned in [this issue](https://github.com/zap-zsh/exa/issues/3), if you are using [zap-zsh/supercharge](https://github.com/zap-zsh/supercharge) you need to load this plugin after *superchage*, otherwise this plugin won't work because *supercharge* define the same `ls` alias (usefull to colorize `ls` output for who is not using **exa**).<BR><BR>
+As mentioned in [this issue](https://github.com/zap-zsh/eza/issues/3), if you are using [zap-zsh/supercharge](https://github.com/zap-zsh/supercharge) you need to load this plugin after *superchage*, otherwise this plugin won't work because *supercharge* define the same `ls` alias (usefull to colorize `ls` output for who is not using **eza**).<BR><BR>
```

Comment:
This line can't be accepted until the name (and so the URL) of this repository is changed.
Only @ChristianChiarulli can do it